test/e2e/basic_empty_dir_test.go

```diff
@@ -1,73 +1,73 @@
-	testframework.MustEnsureImageRegistryIsAvailable(t, client)
-	testframework.MustEnsureInternalRegistryHostnameIsSet(t, client)
-	testframework.MustEnsureClusterOperatorStatusIsSet(t, client)
+// 	cr := &imageregistryapi.ImageRegistry{
+// 		TypeMeta: metav1.TypeMeta{
+// 			APIVersion: imageregistryapi.SchemeGroupVersion.String(),
+// 			Kind:       "ImageRegistry",
+// 		},
+// 		ObjectMeta: metav1.ObjectMeta{
+// 			Name: testframework.ImageRegistryName,
+// 		},
+// 		Spec: imageregistryapi.ImageRegistrySpec{
+// 			OperatorSpec: operatorapi.OperatorSpec{
+// 				ManagementState: operatorapi.Managed,
+// 			},
+// 			Storage: imageregistryapi.ImageRegistryConfigStorage{
+// 				Filesystem: &imageregistryapi.ImageRegistryConfigStorageFilesystem{
+// 					VolumeSource: corev1.VolumeSource{
+// 						EmptyDir: &corev1.EmptyDirVolumeSource{},
+// 					},
+// 				},
+// 			},
+// 			Replicas: 1,
+// 		},
+// 	}
+// 	testframework.MustDeployImageRegistry(t, client, cr)
+// 	testframework.MustEnsureImageRegistryIsAvailable(t, client)
+// 	testframework.MustEnsureInternalRegistryHostnameIsSet(t, client)
+// 	testframework.MustEnsureClusterOperatorStatusIsSet(t, client)
 
-	deploy, err := client.Deployments(testframework.ImageRegistryDeploymentNamespace).Get(testframework.ImageRegistryDeploymentName, metav1.GetOptions{})
-	if err != nil {
-		t.Fatal(err)
-	}
-	if deploy.Status.AvailableReplicas == 0 {
-		testframework.DumpObject(t, "deployment", deploy)
-		t.Errorf("error: the deployment doesn't have available replicas")
-	}
+// 	deploy, err := client.Deployments(testframework.ImageRegistryDeploymentNamespace).Get(testframework.ImageRegistryDeploymentName, metav1.GetOptions{})
+// 	if err != nil {
+// 		t.Fatal(err)
+// 	}
+// 	if deploy.Status.AvailableReplicas == 0 {
+// 		testframework.DumpObject(t, "deployment", deploy)
+// 		t.Errorf("error: the deployment doesn't have available replicas")
+// 	}
 
-	logs, err := testframework.GetOperatorLogs(client)
-	if err != nil {
-		t.Fatal(err)
-	}
-	badlogs := false
-	if !logs.Contains(regexp.MustCompile(`Cluster Image Registry Operator Version: .+`)) {
-		badlogs = true
-		t.Error("error: the log doesn't contain the operator's version")
-	}
-	if !logs.Contains(regexp.MustCompile(`status changed`)) {
-		badlogs = true
-		t.Error("error: the log doesn't contain changes")
-	}
-	if badlogs {
-		testframework.DumpPodLogs(t, logs)
-	}
-}
+// 	logs, err := testframework.GetOperatorLogs(client)
+// 	if err != nil {
+// 		t.Fatal(err)
+// 	}
+// 	badlogs := false
+// 	if !logs.Contains(regexp.MustCompile(`Cluster Image Registry Operator Version: .+`)) {
+// 		badlogs = true
+// 		t.Error("error: the log doesn't contain the operator's version")
+// 	}
+// 	if !logs.Contains(regexp.MustCompile(`status changed`)) {
+// 		badlogs = true
+// 		t.Error("error: the log doesn't contain changes")
+// 	}
+// 	if badlogs {
+// 		testframework.DumpPodLogs(t, logs)
+// 	}
+// }
```

Comment:
why is this entire test being disabled?

Reply:
Doh, didin't mean to commit that, i was just trying to run my test by itself so it was faster locally. fixed in next push